.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
```

Comment:
Per [the Travis docs](https://docs.travis-ci.com/user/docker/#Installing-a-newer-Docker-version), this will need to be `docker-ce`.